plugins/reporters/fossid-snippets/src/main/resources/templates/asciidoc/fossid_snippets.ftl

```diff
@@ -27,10 +27,10 @@
 
 = FossID Snippets
 List of all the packages with their files and snippets.
-[#list ortResult.scanResults as package, scanResults]
+[#assign filteredResults = helper.filterResultsByVCS(ortResult.scanResults)]
```

Comment:
Commit message:
- "*are* two packages"
- I don't get "This is because FossID does not support VCS path and repository is scanned once"
- Please separate the last paragraph by an empty line.